mParticle-Leanplum/MPKitLeanplum.m

```diff
@@ -304,4 +308,11 @@ - (nonnull MPKitExecStatus *)logScreen:(nonnull MPEvent *)event {
     return execStatus;
 }
 
+
+#pragma helper methods
+
+- (FilteredMParticleUser *)currentUser {
+    return [[[MPKitAPI alloc] init] getCurrentUserWithKit:self];
```

Comment:
I believe kitApi is already alloc'd and set on the kitinstance by kit container

Reply:
I might be overly cautious but I added a getter too in case we don't set the property for some reason or codepath.